realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -152,14 +161,25 @@ public boolean isInTransaction() {
      */
     public void addChangeListener(RealmChangeListener listener) {
```

Comment:
We need to highlight somewhere to say that user has to call `removeChangeListener` to avoid memory leak.